packages/path_provider/path_provider_windows/test/path_provider_windows_test.dart

```diff
@@ -7,15 +7,30 @@ import 'dart:io';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:path_provider_platform_interface/path_provider_platform_interface.dart';
 import 'package:path_provider_windows/path_provider_windows.dart';
+import 'package:path_provider_windows/src/path_provider_windows_real.dart'
+    show languageEn, encodingCP1252, encodingUnicode;
 
 // A fake VersionInfoQuerier that just returns preset responses.
 class FakeVersionInfoQuerier implements VersionInfoQuerier {
-  FakeVersionInfoQuerier(this.responses);
+  FakeVersionInfoQuerier(this.responses,
+      [this.language = languageEn, this.encoding = encodingUnicode]);
```

Comment:
Please use named, not positional, arguments for these two optional arguments. (So s/`[]`/`{}`/)

Reply:
+1, makes it easier to read/one less thing to think about at the call-site, which is particularly handy in tests, since they tend to get boilerplate-y.